cluster/gce/util.sh

```diff
@@ -2576,11 +2576,13 @@ function create-cloud-nat-router() {
 }
 
 function delete-all-firewall-rules() {
-  if fws=$(gcloud compute firewall-rules list --project "${NETWORK_PROJECT}" --filter="network=${NETWORK}" --format="value(name)"); then
+  local -a fws
+  kube::util::read-array fws < <(gcloud compute firewall-rules list --project "${NETWORK_PROJECT}" --filter="network=${NETWORK}" --format="value(name)")
+  if (( "${#fws[@]}" > 0 )); then
     echo "Deleting firewall rules remaining in network ${NETWORK}: ${fws}"
```

Comment:
This echo needs to be changed as well, to ${fws[@]} to print more than the first element.

Reply:
Yes, I've noticed it at as. Updated.

Reply:
Actually, that should be `${fws[*]}`
https://tldp.org/LDP/Bash-Beginners-Guide/html/Bash-Beginners-Guide.html#sect_10_02_02